docs/src/main/paradox/communicate-with-actors.md

```diff
@@ -0,0 +1,17 @@
+Communicate with Actors
+-----------------------
+
+All communication with Actors is done through asynchronous message passing. This is what makes Actors reactive and event driven. An Actor doesn't do anything unless it's been told to do something, and we tell it to do something by sending the message. Sending a message asynchronously means that the sender does not stick around waiting for the message to be processed by the recipient actor. Instead, the Actor hands the message off by putting it on the recipient's mailbox and is then free to do something more important than waiting for the recipient to react on the message. The aActor's mailbox is essentially a
```

Comment:
always some confusion about event driven vs message driven, but let's use message driven since that is what reactive manifesto is using